src/pages/index.tsx

```diff
@@ -9,33 +9,28 @@ import Accordion from '@/components/Accordion';
 
 const TASKS_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/tasks`;
 
-const renderCardList = (tasks: task[]) => tasks.map(
-  (item: task) => <Card content={item} key={item.id} />,
-);
+function renderCardList(tasks: task[]) {
```

Comment:
Any reason for shifting away from arrow function approach

Reply:
Rest all LGTM

Reply:
If we would be doing it the other way around the function expression will become a one-liner as per ESLINT standards.
Also, that one-liner will be of length 106 characters. But, ESLINT allows only 100 characters in a line
So changed it to function declaration.

Reply:
Hmmmm, I mean changing the way just for eslint issue is something really new to me.
But ok, you should have thought a lot before coming to this conclusion so cool